### DISCUSSION TITLE
Dynamic Jobs: How to create dynamic timing based on user inputs

### DISCUSSION THREAD

**Discussion by octalpixel** [quirrel-dev/quirrel#1025] [Q&A]
I have a use case where I want to do scheduled jobs based on what the user wants to set. Is this currently possible with Quirrel? If so how can I achieve this natively.

**Comment by Skn0tt** [comment]
Take a look at the `runAt` parameter: https://docs.quirrel.dev/api/queue#runat

**Reply by octalpixel** [reply]
What if there are multiple users?  Would it be overwriting it every time? Couldn't find much details on the doc (or did I miss it)

**Reply by Skn0tt** [reply]
For every `.enqueue` call, it would create a job (unless you use the `override` parameter).
Maybe you're confusing `Queue` with `CronJob`?
